offer-manager-domain/src/main/groovy/life/qbic/business/products/copy/CopyProduct.groovy

```diff
@@ -12,9 +15,68 @@ import life.qbic.datamodel.dtos.business.ProductId
  * @since: 1.0.0
  *
  */
-class CopyProduct implements CopyProductInput {
+class CopyProduct implements CopyProductInput, CreateProductOutput {
```

Comment:
The CopyProduct use case should not implement the CreateProductOutput but rather the CreateProduct use case should implement the CopyProductOutput

Reply:
No because I need to do logic in the copy use case depending on the output of the create use case. Regardless please ignore the copy use case files in this PR.